src/components/sbb-card/sbb-card.scss

```
@@ -0,0 +1,76 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  position: relative;
+  background-color: var(--sbb-color-white-default);
+  width: 100%;
+  border-radius: var(--sbb-border-radius-4x);
+  padding-inline: var(--sbb-spacing-responsive-xxs-small);
+  overflow: hidden;
+  transition-property: box-shadow var(--sbb-animation-duration-4x) var(--sbb-animation-easing);
```

```suggestion
  transition: box-shadow var(--sbb-animation-duration-4x) var(--sbb-animation-easing);
```
Otherwise it can not work?

---

@jeripeierSBB 
It works; I checked Figma again and I see that from default to hover the transition duration is 30ms.
We don't have this token, so I put for now --sbb-animation-duration-6x, which is 24ms.
Should I add this new value in lyne-design-token?

---

maybe during rendering it will be corrected automatically ;) but it should be  "transition:". yes take the 24ms and tell manuel to update it in figma. with this little difference there is no need to add a new token.

---

Done ;)